storage/rocksdb/rdb_native_dd.cc

```diff
@@ -0,0 +1,61 @@
+/*
+   Copyright (c) 2023 Meta, Inc
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA */
+
+/* This C++ file's header file */
+#include "rdb_native_dd.h"
+
+/* MySQL header files */
+#include "sql/dd/types/table.h"  // dd::Table
+
+/* MyRocks header files */
+#include "ha_rocksdb.h"
+
+namespace myrocks {
+void rocksdb_dict_register_dd_table_id(dd::Object_id dd_table_id) {
+  native_dd::insert_dd_table_ids(dd_table_id);
+};
+
+bool ha_rocksdb::get_se_private_data(dd::Table *, bool reset) {
```

Comment:
could you implement this into ha_rocksdb.cc?  so that all ha_rocksdb class implemented in one place?